src/poliastro/examples.py

```diff
@@ -16,12 +16,14 @@
     [7.37289205, 2.08223573, 4.39999794e-1] * u.km / u.s,
     time.Time("2013-03-18 12:00", scale='utc')
 )
+'''ISS orbit example'''
```

Comment:
Yes, they appear, altough I only wrote those docstrings because the examples.py module appeared without any text in the API Reference.